src/style.css

```
@@ -1 +1,63 @@
 /* Styles go here */
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+
+  list-style-type: none;
+
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 20px;
+}
```

you allready add margin to all .nav__item.
```suggestion
.nav__item {
  display: flex;
  margin-right: 20px;
}
```

or do you need it for some purpose?